docs/notebooks/gis.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p>Idem, il faudrait ajouter quelques informations de base sur d'où viennent les données.</p><p> </p><p><code>Land use classification is powered by the Planetary Computer's STAC catalog. It uses the '10m Annual Land Use Land Cover' dataset by default ("io-lulc-9-class"), but other collections can be specified by using the collection argument. </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>